roles/install_agent/vars/main.yml

```diff
@@ -1,9 +1,9 @@
 ---
 # vars file for install_agent
-pkg_format: "{% if ansible_facts.pkg_mgr in ['yum', 'dnf', 'zypper'] %}rpm{%
-  elif ansible_facts.pkg_mgr == 'apt' %}deb{%
-  elif ansible_facts.os_family == 'Windows' and not win_use_exe %}msi{%
-  elif ansible_facts.os_family == 'Windows' and win_use_exe %}exe{%
+pkg_format: "{% if ansible_facts.pkg_mgr | default([]) in ['yum', 'dnf', 'zypper'] %}rpm{%
```

Comment:
@jfunke4 the type of pkg_mgr variable is a string not a list

Reply:
Your right. We will fix this in a future release. 
For now it's important to have this var defined, as it is not included in the ansible facts on windows systems

Reply:
I see the problem. Thanks for fixing it!